chart/values.yaml

```diff
@@ -98,13 +98,13 @@ keepDNSUnchanged:
   # -- Keep existing DNS configuration
   enabled: false
 
-configFiles:
-  # -- The OpenVPN config file to use. If this is unset, the first file with the extension `.conf` will be used.
-  openVPNConfig: ""
+vpnConfig:
+  existingSecret: ""
```

Comment:
Meta comment, HELM charts are quite hard to review, this can be simplified by having two commits:
- one commit where you move everything under `vpnConfig`
- one commit where you add `existingSecret`
No need to change that now, but keep in mind for the future